calico-enterprise/operations/ebpf/install.mdx

```diff
@@ -35,9 +35,10 @@ and in particular, pushing the networking capabilities of the latest Linux kerne
 
 - Linux distribution/kernel:
 
-  - Ubuntu 20.04 and 22.04.
+  - Ubuntu 20.04 or above.
   - Red Hat v8.2 with Linux kernel v4.18.0-193 or above (Red Hat have backported the required features to that build).
   - Another [supported distribution](../../getting-started/install-on-clusters/requirements.mdx) with Linux kernel v5.3 or above.
```

Comment:
unrelated to this PR, this seems like pretty old stuff. 20.04 is no longer supported bu canonical, RedHat 8.2 is no longer supported as well and I think we should bump up 5.3 to something more reasonable.

We for instance know that ubuntu 22.04 with 5.15 no longer works for us. Clouds still distribute it, but 22.04 no mostly comes with 6.8

Seems like this was relevant 5+ years ago when there we kernels that did not have ebpf support. We still kinda support these old ones (in the sense that we should run there with some degradation perhaps), but we do not really test it anymore.